src/ethereum/EtherscanButton/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { Icon } from '../..';
+
+type Props = {
+  className?: string;
+  network?: 'mainnet' | 'rinkeby';
+  type: 'address' | 'tx';
+  value: string;
+};
+
+const EtherscanButton = ({
+  className,
+  type,
+  value,
+  network = 'mainnet',
+}: Props): React.ReactElement => {
+  const getNetwork = () => {
+    const lowerCaseNetwork = network.toLowerCase();
+    return lowerCaseNetwork === 'mainnet' ? '' : `${lowerCaseNetwork}.`;
+  };
+
+  const goToEtherscan = () =>
```

Comment:
this function name has nothing to do with what it performs, it doesn't move you anywhere, it just returns an etherscan link. Also, it could be defined outside of the component like it is done in safe-react